packages/manager/modules/netapp/src/dashboard/index/template.html

```diff
@@ -43,13 +43,19 @@
                 <oui-tile-description>
                     <span
                         class="oui-badge"
+                        data-ng-if="!$ctrl.networkInformations.vrackServices || $ctrl.networkInformations.vrackServices.currentState.productStatus !== $ctrl.VRACK_SERVICES_STATUS.DISABLED"
```

Comment:
Is $ctrl.networkInformations.vrackServices.currentState.productStatus !== $ctrl.VRACK_SERVICES_STATUS.DISABLED necessary ?
It seems that disabled services are filtered in the routing

Reply:
The filter in the routing is for network configuration page not dashboard page